ludwig/benchmarking/benchmark.py

```diff
@@ -77,5 +69,20 @@ def benchmark(bench_config_path: str) -> None:
                 experiment["experiment_name"] = config["global_experiment_name"]
             benchmark_one_local(experiment, export_artifacts_dict=config["export"][0])
         except Exception:
-            print("Benchmarking {} {} failed".format(experiment["dataset_name"], experiment["experiment_name"]))
-            print(traceback.format_exc())
+            logging.exception(
+                "Benchmarking {} {} failed".format(experiment["dataset_name"], experiment["experiment_name"])
+            )
+
+
+def cli(sys_argv):
+    parser = argparse.ArgumentParser(
+        description="This script runs a ludwig experiment on datasets specified in the benchmark config and exports "
+        "the experiment artifact for each of the datasets following the export parameters specified in"
+        "the benchmarking config.",
+        prog="ludwig benchmark",
+        usage="%(prog)s [options]",
+    )
+    parser.add_argument("--config", type=str, help="The benchmarking config.")
```

Comment:
Maybe let's use the argument `benchmarking_config` as to not conflate this with a regular ludwig config.